lbry/blockchain/lbrycrd.py

```diff
@@ -243,6 +243,8 @@ def unsubscribe(self):
             self.subscription = None
 
     async def rpc(self, method, params=None):
+        if self.session.closed:
+            raise Exception("session is closed! We are shutting down.")
```

Comment:
who's we

Reply:
changed to explain the real issue: `session is closed! RPC attempted during shutting down.`